livekit-android-sdk/src/main/java/io/livekit/android/e2ee/E2EEManager.kt

```diff
@@ -122,6 +135,19 @@ constructor(keyProvider: KeyProvider) {
         }
     }
 
+    public fun removePublishedTrack(track: Track, publication: TrackPublication, participant: LocalParticipant, room: Room) {
+        for (item in frameCryptors.entries) {
+            var key = item.key
+            var frameCryptor = item.value
+            if (key.first == participant.sid) {
```

Comment:
I think this should be `key.second`? trackId is in the first position and participantId in the second position.

Reply:
okay, I found the error, this should be `publication.sid` not `participant.sid`, which is being verified.